packages/client/tests/functional/driver-adapters/_utils/test-suite-options.ts

```diff
@@ -4,9 +4,9 @@ import { MatrixOptions } from '../../_utils/types'
  * Opt out from providers not specified in `driverAdaptersTestProviders` in `setupTestSuite`
  * (to avoid the error about missing providers in the test suite matrix).
  */
-export const optOutFromProvidersWithNoMatchingFlavours: MatrixOptions['optOut'] = {
+export const optOutFromProvidersWithNoMatchingDriverAdapters: MatrixOptions['optOut'] = {
   from: ['cockroachdb', 'sqlserver', 'mongodb'],
-  reason: 'no corresponding flavour of driver adapters yet',
+  reason: 'no availability of a Driver Adapter with these providers yet',
```

Comment:
Off topic for this PR, but technically we could run cockroachdb through pg or neon adapter I think

Reply:
Possibly, but only if the PSL `provider` is `postgres`, not `cockroachdb`.